src/main/resources/swagger.api/inn-reach.yaml

```diff
@@ -5,175 +5,161 @@ info:
       requestBody:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/moduleConfiguration"
+              $ref: "#/components/schemas/innReachLocationDTO"
         required: true
-    put:
+    get:
       responses:
-        '201':
-          description: ""
+        '200':
+          description: "OK"
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/moduleConfiguration"
-        '400':
-          description: Bad request
+                $ref: "#/components/schemas/innReachLocationsDTO"
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+      description: Get a list of InnReach locations
+      operationId: getLocations
+      parameters:
+        - $ref: "#/components/parameters/trait_pageable_offset"
+        - $ref: "#/components/parameters/trait_pageable_limit"
+  /locations/{locationId}:
+    get:
+      responses:
+        '200':
+          description: OK
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Bad request
+                $ref: "#/components/schemas/innReachLocationDTO"
         '404':
-          description: Configuration not found
+          description: InnReach location not found
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Configuration not found
+                $ref: "#/components/schemas/error"
         '500':
           description: Internal server error
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Internal server error
-      description: Change the inn reach configuration
-      operationId: putConfiguration
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: "#/components/schemas/moduleConfiguration"
-        required: true
-  /configurations/{configId}:
-    get:
+                $ref: "#/components/schemas/error"
+      description: Get InnReach location by id
+      operationId: getLocationById
+      parameters:
+        - $ref: '#/components/parameters/locationId'
+    put:
       responses:
         '200':
-          description: ""
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/innReachLocationDTO"
+        '400':
+          description: Bad request
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/moduleConfiguration"
+                $ref: "#/components/schemas/validationErrorsDTO"
```

Comment:
The same here: as mentioned for Locations DTO above, do we need to use this notation when define entity's schema?